server/etcdmain/grpc_proxy.go

```diff
@@ -538,8 +538,7 @@ func mustHTTPListener(lg *zap.Logger, m cmux.CMux, tlsinfo *transport.TLSInfo, c
 func mustNewHTTPClient(lg *zap.Logger) *http.Client {
 	transport, err := newHTTPTransport(grpcProxyCA, grpcProxyCert, grpcProxyKey)
 	if err != nil {
-		fmt.Fprintln(os.Stderr, err)
-		os.Exit(1)
+		lg.Fatal("failed to new http transport", zap.Error(err))
```

Comment:
This is not the same, it changes the exit code and the output.

Reply:
`lg.Fatal` exits with code 1. Yes, I should not update the output.
Updated with removing the `lg *zap.Logger` since no function uses `lg`.